.github/workflows/codeql.yml

```diff
@@ -0,0 +1,81 @@
+name: "CodeQL"
```

Comment:
do you have a preview of this job's execution?

Reply:
Here is a sample execution of this workflow: https://github.com/PatKamin/unified-runtime/actions/runs/5011460219/jobs/8982314529